src/pcapi/core/offers/api.py

```diff
@@ -541,3 +551,12 @@ def deactivate_inappropriate_product(isbn: str) -> bool:
             redis.add_offer_id(client=app.redis_client, offer_id=offer_id)
 
     return True
+
+
+def compute_offer_validation(offer: Offer) -> OfferValidationStatus:
```

Comment:
À la première lecture on imagine un vrais calcule, ca me semble bien de spécifier ce que fait la fonction ici:

```suggestion
def compute_offer_validation_from_name(offer: Offer) -> OfferValidationStatus:
```